src/components/Content/Pursuance/views/DiscussView.css

```diff
@@ -61,8 +61,7 @@
   display: flex;
   align-items: center;
   justify-content: space-between;
-  padding: 0 0;
-  margin: 0 -6px 0 0;
+  padding: 0 7px;
```

Comment:
@Moe-Shoman Was this 7px and not 8px?...  I'll `git checkout` locally now.

Reply:
Done

Reply:
I believe it was 7px after pulling your changes to create a 30*30 box